docs/configuration.md

```diff
@@ -746,6 +746,13 @@ Apart from these, the following properties are also available, and may be useful
     *Warning*: This will increase the size of the event log considerably.
   </td>
 </tr>
+<tr>
+  <td><code>spark.eventLog.longForm.enabled</code></td>
+  <td>false</td>
+  <td>
+    Whether to use the long form of call sites in the event log.
```

Comment:
This seems fine. If you have to change anything, I might say this as "If true, use the long form of call sites in the event log. Otherwise use the short form." Just to clarify what long form is an alternative to.